src/components/rules-list-page/SyncRulesPage.jsx

```diff
@@ -42,35 +43,44 @@ class SyncRulesPage extends React.Component {
         syncReport: SyncReport.create(),
         syncSummaryOpen: false,
     };
-    initialSorting = ["name", "asc"];
 
-    getValueForTargetInstances = ruleData => {
-        const { allInstances } = this.state;
-        const rule = SyncRule.build(ruleData);
-        return getValueForCollection(
-            rule.targetInstances
-                .map(id => allInstances.find(instance => instance.id === id))
-                .map(({ name }) => ({ name }))
-        );
-    };
+    initialSorting = ["name", "asc"];
 
     columns = [
         { name: "name", text: i18n.t("Name"), sortable: true },
         {
             name: "targetInstances",
             text: i18n.t("Destination instances"),
             sortable: false,
-            getValue: this.getValueForTargetInstances,
+            getValue: ruleData => {
+                const { allInstances } = this.state;
+                const rule = SyncRule.build(ruleData);
+                return _(rule.targetInstances)
+                    .map(id => allInstances.find(instance => instance.id === id))
+                    .compact()
+                    .map(({ name }) => name)
+                    .join(", ");
+            },
         },
     ];
+
     detailsFields = [
         { name: "name", text: i18n.t("Name") },
         { name: "description", text: i18n.t("Description") },
         {
             name: "targetInstances",
             text: i18n.t("Destination instances"),
             sortable: true,
-            getValue: this.getValueForTargetInstances,
+            getValue: ruleData => {
+                const { allInstances } = this.state;
+                const rule = SyncRule.build(ruleData);
+                return getValueForCollection(
+                    _(rule.targetInstances)
+                        .map(id => allInstances.find(instance => instance.id === id))
+                        .compact()
+                        .map(({ name }) => ({ name }))
+                );
+            },
```

Comment:
Those two `getValue` share some code. Maybe we can abstract it to a method. Something like: 

```
            getTargetInstances() {
                const { allInstances } = this.state;
                const rule = SyncRule.build(ruleData);
                return _(rule.targetInstances)
                    .map(id => allInstances.find(instance => instance.id === id))
                    .compact()
                    .map(({ name }) => ({ name }))
                );
            }
```

Reply:
Yeah sounds good